R/on-load.R

```diff
@@ -3,4 +3,9 @@
 
 .onLoad <- function(libname, pathname) {
   debugme()
+
+  shiny::addResourcePath(
+    "shinytest",
+    system.file("js", package = "shinytest")
+  )
```

Comment:
Good catch. I'll move into `with_shinytest_js`. There's no way in shiny to remove a resource path, so I guess there will a side-effect for calling `with_shinytest_js()` in a normal R session, but that shouldn't hurt anything.